examples/traylayout.txt

```diff
@@ -1,13 +1,13 @@
- ,> 40.0mm
- | ,> 50.0mm
- | | ,> 50.0mm
- | | | ,> 40.0mm
+ ,> 0.0mm
+ | ,> 0.0mm
+ | | ,> 0.0mm
+ | | | ,> 0.0mm
 +-+-+-+-+
-|   |   | 60.0mm
```

Comment:
Hello! I'm wondering, are these value changes from examples deliberately part of this pull request?

Reply:
Yeah, this needs to be reverted and the SVG re-created.

Reply:
`traylayout.txt` is output of the generator. Seems the generator only outputs the layout without any values. Need some more investigation how to change the test.